oarepo_ui/templates/oarepo_ui/header_login.html

```diff
@@ -21,6 +21,47 @@
             {% endif %}
         </form>
   {%- else -%}
+
+      {# Desktop Quick-create ("plus") menu #}
+    {%- set plus_menu_items = current_menu.submenu('plus').children %}
+    {%- if plus_menu_items %}
+      <div
+        id="quick-create-dropdown"
+        class="rdm-plus-menu rdm-plus-menu-responsive ui dropdown pr-15 computer only"
```

Comment:
Do we have these rdm* classes defined in oarepo-ui?

Reply:
@mirekys We don't, you are right, I will remove it (it is something used in RDM). Sorry!